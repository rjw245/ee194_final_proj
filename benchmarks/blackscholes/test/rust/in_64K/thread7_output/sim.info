{'cmdline': ['/h/hzhou01/comp140/sniper/run-sniper',
             '-d',
             './thread7_output',
             '-n',
             '8',
             '-c',
             'gainestown',
             '-c',
             'nehalem.cfg',
             '--',
             '../../../rust/bs',
             '7',
             '../../../inputs/in_64K.txt'],
 'git_revision': '',
 'host': 'dell24',
 'pin_version': 71313,
 'rusage': (54616.580017,
            721.705284,
            175664,
            0,
            0,
            0,
            441984,
            0,
            0,
            0,
            12448,
            0,
            0,
            0,
            47783620,
            162184),
 'snipercmd': ['bash',
               '-c',
               '/h/hzhou01/comp140/sniper/pin_kit/intel64/bin/pinbin -mt -injection child -xyzzy -enable_vsm 0 -t /h/hzhou01/comp140/sniper/lib/pin_sim -c /h/hzhou01/comp140/sniper/config/base.cfg --general/total_cores=8 --general/output_dir=/h/hzhou01/ee194_final_proj/benchmarks/blackscholes/test/rust/in_64K/thread7_output --config=/h/hzhou01/comp140/sniper/config/nehalem.cfg --config=/h/hzhou01/comp140/sniper/config/gainestown.cfg --config=/h/hzhou01/comp140/sniper/config/nehalem.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -- ../../../rust/bs 7 ../../../inputs/in_64K.txt'],
 't_elapsed': 21911.501695871353,
 't_start': 1462232497.532021,
 'tracecmds': [],
 'user': 'hzhou01',
 'vmem': 4025606144L}
