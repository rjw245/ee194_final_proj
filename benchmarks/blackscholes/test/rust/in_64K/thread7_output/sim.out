                                     | Core 0      | Core 1      | Core 2      | Core 3      | Core 4      | Core 5      | Core 6      | Core 7     
  Instructions                       |  4219259076 |  1901214536 |  1901237872 |  1901261874 |  1901231310 |  1901220352 |  1901267381 |  1901216824
  Cycles                             |  4116699258 |  4116699258 |  4116699258 |  4116699258 |  4116699258 |  4116699258 |  4116699258 |  4116699258
  IPC                                |        1.02 |        0.46 |        0.46 |        0.46 |        0.46 |        0.46 |        0.46 |        0.46
  Time (ns)                          |  1547631300 |  1547631300 |  1547631300 |  1547631300 |  1547631300 |  1547631300 |  1547631300 |  1547631300
  Idle time (ns)                     |   610200608 |   937412857 |   951851151 |   951664895 |   951327212 |   947663201 |   951442521 |   939091291
  Idle time (%)                      |       39.4% |       60.6% |       61.5% |       61.5% |       61.5% |       61.2% |       61.5% |       60.7%
Branch predictor stats               |             |             |             |             |             |             |             |            
  num correct                        |   148048537 |    66896521 |    67253159 |    67253576 |    67241327 |    67022115 |    67250538 |    66918028
  num incorrect                      |     6096400 |     2082989 |     1726500 |     1725770 |     1737664 |     1955813 |     1729271 |     2060668
  misprediction rate                 |       3.95% |       3.02% |       2.50% |       2.50% |       2.52% |       2.84% |       2.51% |       2.99%
  mpki                               |        1.44 |        1.10 |        0.91 |        0.91 |        0.91 |        1.03 |        0.91 |        1.08
TLB Summary                          |             |             |             |             |             |             |             |            
  I-TLB                              |             |             |             |             |             |             |             |            
    num accesses                     |   903864692 |   312100516 |   312103327 |   312104290 |   312101749 |   312098430 |   312105645 |   312101346
    num misses                       |         199 |          76 |          76 |          71 |          71 |          72 |          71 |          71
    miss rate                        |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  D-TLB                              |             |             |             |             |             |             |             |            
    num accesses                     |  2813246433 |  1093066781 |  1093080438 |  1093099802 |  1093081033 |  1093073093 |  1093105142 |  1093068972
    num misses                       |      342882 |        4617 |        3271 |        4317 |        3827 |        3732 |        4419 |        3236
    miss rate                        |       0.01% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |        0.08 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  L2 TLB                             |             |             |             |             |             |             |             |            
    num accesses                     |      343081 |        4693 |        3347 |        4388 |        3898 |        3804 |        4490 |        3307
    num misses                       |         837 |         260 |         237 |         215 |         204 |         216 |         207 |         208
    miss rate                        |       0.24% |       5.54% |       7.08% |       4.90% |       5.23% |       5.68% |       4.61% |       6.29%
    mpki                             |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
Cache Summary                        |             |             |             |             |             |             |             |            
  Cache L1-I                         |             |             |             |             |             |             |             |            
    num cache accesses               |   903864692 |   312100516 |   312103327 |   312104290 |   312101749 |   312098430 |   312105645 |   312101346
    num cache misses                 |    22987618 |        5149 |        6668 |        6802 |        7401 |        6515 |        6925 |        5701
    miss rate                        |       2.54% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |        5.45 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  Cache L1-D                         |             |             |             |             |             |             |             |            
    num cache accesses               |  2813247170 |  1093067036 |  1093080720 |  1093100104 |  1093081332 |  1093073413 |  1093105471 |  1093069315
    num cache misses                 |      103412 |      393745 |      394944 |      394700 |      394820 |      395175 |      397171 |      392916
    miss rate                        |       0.00% |       0.04% |       0.04% |       0.04% |       0.04% |       0.04% |       0.04% |       0.04%
    mpki                             |        0.02 |        0.21 |        0.21 |        0.21 |        0.21 |        0.21 |        0.21 |        0.21
  Cache L2                           |             |             |             |             |             |             |             |            
    num cache accesses               |    23091207 |      398915 |      401622 |      401519 |      402253 |      401725 |      404122 |      398630
    num cache misses                 |       39277 |       40220 |       53833 |       54902 |       61051 |       55766 |       64723 |       46146
    miss rate                        |       0.17% |      10.08% |      13.40% |      13.67% |      15.18% |      13.88% |      16.02% |      11.58%
    mpki                             |        0.01 |        0.02 |        0.03 |        0.03 |        0.03 |        0.03 |        0.03 |        0.02
  Cache L3                           |             |             |             |             |             |             |             |            
    num cache accesses               |       39468 |       40241 |       53843 |       54919 |       61083 |       55801 |       64749 |       46159
    num cache misses                 |       33939 |         690 |         531 |         460 |        4752 |        3998 |        4197 |        3951
    miss rate                        |      85.99% |       1.71% |       0.99% |       0.84% |       7.78% |       7.16% |       6.48% |       8.56%
    mpki                             |        0.01 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
DRAM summary                         |             |             |             |             |             |             |             |            
  num dram accesses                  |       18436 |           0 |           0 |           0 |       18435 |           0 |           0 |           0
  average dram access latency (ns)   |       59.87 |         inf |         inf |         inf |       59.42 |         inf |         inf |         inf
  average dram queueing delay        |        6.44 |        0.00 |        0.00 |        0.00 |        5.99 |        0.00 |        0.00 |        0.00
  average dram bandwidth utilization |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
Coherency Traffic                    |             |             |             |             |             |             |             |            
  num loads from dram                |        3854 |         146 |          69 |          59 |          38 |          38 |          40 |          38
  num loads from dram cache          |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  num loads from remote cache        |          29 |          44 |          43 |          55 |        3717 |        3600 |        3603 |        3574
